src/unstract/sdk/adapters/utils.py

```diff
@@ -25,14 +28,20 @@ def get_msg_from_request_exc(
         Returns:
             str: Error message returned by the server
         """
-        if hasattr(err, "response"):
-            err_response: Response = err.response  # type: ignore
-            if err_response.headers["Content-Type"] == "application/json":
-                err_json = err_response.json()
-                if message_key in err_json:
-                    return str(err_json[message_key])
-            elif err_response.headers["Content-Type"] == "text/plain":
-                return err_response.text  # type: ignore
+        if not hasattr(err, "response"):
+            return default_err
+        err_response: Response = err.response  # type: ignore
+        if err_response.headers["Content-Type"] == "application/json":
+            err_json = err_response.json()
+            if message_key in err_json:
+                return str(err_json[message_key])
+            else:
+                logger.warning(
+                    f"Unable to parse error with key '{message_key}' for "
+                    f"'{err_json}', returning '{default_err}' instead."
+                )
+        elif err_response.headers["Content-Type"] == "text/plain":
+            return err_response.text  # type: ignore
```

Comment:
@chandrasekharan-zipstack making sure we intend to return deafult error if it is the else condition right? May be a good idea to add the else since we do not have any other code here

Reply:
Will explicitly add the else